spark-connector/spark-common/src/main/java/com/datastrato/gravitino/spark/connector/plugin/GravitinoDriverPlugin.java

```diff
@@ -58,6 +62,8 @@ public Map<String, String> init(SparkContext sc, PluginContext pluginContext) {
         String.format(
             "%s:%s, should not be empty", GravitinoSparkConfig.GRAVITINO_METALAKE, metalake));
 
+    gravitinoDriverExtensions.addAll(toRegisteredDriverExtensions);
```

Comment:
GravitinoIcebergSparkSessionExtensions should enabled only if  `enableIcebergSupport` is true.

Reply:
fixed